Puppetfile

```diff
@@ -0,0 +1,29 @@
+moduledir 'development/modules'
+
+mod 'puppetlabs-stdlib',
+  :git => 'git@github.com:puppetlabs/puppetlabs-stdlib.git',
+  :tag => '4.6.0'
+
+mod 'puppetlabs-registry',
+  :git => 'git@github.com:puppetlabs/puppetlabs-registry.git',
+  :tag => '1.0.0'
+
+mod 'puppetlabs-acl',
+  :git => 'git@github.com:puppetlabs/puppetlabs-acl.git',
+  :tag => '2.0.1'
+
+mod 'puppetlabs-powershell',
+  :git => 'git@github.com:puppetlabs/puppetlabs-powershell.git',
+  :tag => '1.0.4'
+
+mod 'puppetlabs-iis',
+  :git => 'git@github.com:puppetlabs/puppetlabs-iis.git',
+  :tag => '4.1.0'
+
+mod 'puppet-windowsfeature',
+  :git => 'git@github.com:voxpupuli/puppet-windowsfeature.git',
+  :tag => 'v1.0.0'
```

Comment:
This is no longer a dependency, please remove.

Reply:
Or is it a transient dependency for something else?

Reply:
This is a transient dependency. I used the `Puppetfile` to bring down runtime dependencies so vagrant can use them.  Once we figure out the travis-ci feature I guess the whole Puppetfile will be gone as well as the Vagrantfile.